src/compiler/crystal/codegen/llvm_typer.cr

```diff
@@ -462,15 +462,15 @@ module Crystal
     end
 
     # Copy existing LLVM types, possibly from another context,
-    # into this typer's context.
+    # into this type's context.
     def copy_types(types : Array(LLVM::Type))
       types.map do |type|
         copy_type(type).as(LLVM::Type)
       end
     end
 
     # Copy an existing LLVM type, possibly from another context,
-    # into this typer's context.
+    # into this type's context.
```

Comment:
ditto

Reply:
Ah indeed, I was thinking about this but wasn't sure. The two occurrences you mention are the only ones such. Will update, thanks!